### DISCUSSION TITLE
I need someone real to re-verify me

### DISCUSSION THREAD

**Discussion by igorrates** [community/community#50844] [GitHub Education]
I tried providing everything for GitHub that proves I'm a student. School ID, School Portal screenshot, class schedule, and so on but they are asking for a school email but my school doesn't provide an email for the students.

**Comment by samus-aran** [comment]
Hi @igorrates  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
